interop-testing/src/main/java/io/grpc/testing/integration/TestServiceClient.java

```diff
@@ -290,16 +290,6 @@ private void runTest(TestCases testCase) throws Exception {
     @Override
     protected ManagedChannel createChannel() {
       if (!useOkHttp) {
-        InetAddress address;
-        try {
-          address = InetAddress.getByName(serverHost);
-          if (serverHostOverride != null) {
-            // Force the hostname to match the cert the server uses.
-            address = InetAddress.getByAddress(serverHostOverride, address.getAddress());
-          }
-        } catch (UnknownHostException ex) {
-          throw new RuntimeException(ex);
-        }
```

Comment:
Errr... at least, I'm not trying to make `forAddress(InetSocketAddress)` work with proxies. Since it avoids the NameResolver.